roofit/roofitZMQ/CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ ROOT_LINKER_LIBRARY(RooFitZMQ
     )
 
 target_link_libraries(RooFitZMQ PUBLIC libzmq cppzmq)
-set(RooFitZMQ_INCLUDE_DIRS ${CMAKE_CURRENT_SOURCE_DIR}/res)
+set(RooFitZMQ_INCLUDE_DIRS ${CMAKE_CURRENT_SOURCE_DIR}/res CACHE INTERNAL "")
```

Comment:
Same as the other one, this should not be set in the cache.

Reply:
See conversation above.